spec/models/employee_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'rails_helper'
+
+RSpec.describe Employee, type: :model do
+  before :all do
+    @project = Project.create(title: "a", description: "b", due_date: Time.now + 3.days)
+    @pm = ProjectManager.create(email: "example30@example.com",
```

Comment:
Generally I would set up factory bot with a unique sequence / Faker / etc. and other testing libraries but for now just using non-repeating emails